index.src.html

```diff
@@ -1233,6 +1233,36 @@ <h3 id="intervention-report">Intervention</h3>
       column number in [=InterventionReportBody/sourceFile=] of the offending
       behavior (which prompted the intervention), or null otherwise.
 
+  <h3 id="crash-report">Crash</h3>
```

Comment:
nit: “Crash reports” for the section title

Reply:
"crash-report" is in line with the other section ids so far.

Reply:
Yep yep, I meant the human-readable part, not the `id`

Reply:
Oh! I misunderstood, although that is also in line with "Deprecation" and "Intervention" titles. I thought that made sense without "Reports" being under the "Report Types" header. Do you think they should all be changed?

Reply:
Ah right!  Nah, keep as-is